### PR TITLE
Improve MDATP run-analyzer-macos-linux.md

### PR DESCRIPTION
**Pull request by rico-chet**
- Fix inconsistent zip file hash sum
- Unzip to a subdirectory to avoid clutter
- Fix typos
- Add `sudo` where the privileges are needed
- Remove useless "GUI" scenario
- Add hash sum check command
- Reformat for easier copying of steps' commands

Tested to work on a Debian 11 on 2023-01-18.

### CODE REVIEW

**Comment by rico-chet** [comment]
@microsoft-github-policy-service agree [company="{Siemens Inc}"]

**Comment by rico-chet** [comment]
@microsoft-github-policy-service agree company="Siemens Inc"

File: microsoft-365/security/defender-endpoint/run-analyzer-macos-linux.md [hunk 1]
```diff
@@ -28,47 +29,56 @@ search.appverid: met150
...
+
+```sh
+cd XMDEClientAnalyzer
+```
 
-1. `wget --quiet -O XMDEClientAnalyzer.zip https://aka.ms/XMDEClientAnalyzer`
+### Install the components
 
-2. `unzip -q XMDEClientAnalyzer.zip`
+Run as non-root user to install required pip and lxml components.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/run-analyzer-macos-linux.md]:
```suggestion
Run as a non-root user to install required pip and lxml components.
```

File: microsoft-365/security/defender-endpoint/run-analyzer-macos-linux.md [hunk 2]
```diff
@@ -28,47 +29,56 @@ search.appverid: met150
...
-3. `cd XMDEClientAnalyzer`
+```sh
+./mde_support_tool.sh
+```
 
-4. `chmod +x mde_support_tool.sh`
+### Collect the diagnosics
 
-3. Run as non-root use to install required pip and lxml which components: `./mde_support_tool.sh`
+To collect actual diagnostic package and generate the result archive file, run again as root.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/run-analyzer-macos-linux.md]:
```suggestion
To collect the actual diagnostic package and generate the result archive file, run again as root.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rico-chet, kindly commit the minor suggested changes. Many thanks.

**Comment by rico-chet** [comment]
> Copy review - @rico-chet, kindly commit the minor suggested changes. Many thanks.

done, you're welcome!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@rico-chet please pass all the checks. Thanks

**Comment by rico-chet** [comment]
> @rico-chet please pass all the checks. Thanks

I rebased the branch to re-trigger the build but the **OpenPublishing.Build** check is again in the state _Expected — Waiting for status to be reported_ and I don't have a hint or clue what to do.

**Pull request merged**